x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/find/find_gaps_route.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { IRouter } from '@kbn/core/server';
+import {
+  findGapsBodySchemaV1,
+  FindGapsRequestBodyV1,
+  FindGapsResponseV1,
+} from '../../../../../common/routes/gaps/apis/find';
+import { ILicenseState } from '../../../../lib';
+import { verifyAccessAndContext } from '../../../lib';
+import {
+  AlertingRequestHandlerContext,
+  INTERNAL_ALERTING_GAPS_FIND_API_PATH,
+} from '../../../../types';
+import { transformRequestV1, transformResponseV1 } from './transforms';
+
+export const findGapsRoute = (
```

Comment:
could we rename this `findGapsForRule` since rule ID is a required parameter? Would make it easier to differentiate this from `findGapsForId`